source/utilities.tex

```diff
@@ -1357,18 +1427,17 @@
   template<class... Types>
     class tuple;
 
-  template<class... TTypes, class... UTypes, template<class> class TQual,
-           template<class> class UQual>
-    requires requires { typename tuple<common_reference_t<TQual<TTypes>, UQual<UTypes>>...>; }
-  struct basic_common_reference<tuple<TTypes...>, tuple<UTypes...>, TQual, UQual> {
-    using type = tuple<common_reference_t<TQual<TTypes>, UQual<UTypes>>...>;
-  };
+  template <typename T>
+    concept @\defexposconcept{tuple-like}@ = @\seebelownc@;         // \expos
+  template <typename T>
+    concept @\defexposconcept{pair-like}@ =                     // \expos
+      @\exposconcept{tuple-like}@<T> && tuple_size_v<remove_cvref_t<T>> == 2;
 
-  template<class... TTypes, class... UTypes>
-    requires requires { typename tuple<common_type_t<TTypes, UTypes>...>; }
-  struct common_type<tuple<TTypes...>, tuple<UTypes...>> {
-    using type = tuple<common_type_t<TTypes, UTypes>...>;
-  };
+  template<@\exposconcept{tuple-like}@ TTuple, @\exposconcept{tuple-like}@ UTuple,
```

Comment:
```suggestion
  // \ref{tuple.common.ref}, \tcode{common_reference}-related specializations
  template<@\exposconcept{tuple-like}@ TTuple, @\exposconcept{tuple-like}@ UTuple,
```

Reply:
Added, but "`common_reference`-related" isn't a term, so didn't include the "-".